vendor/vendor.json

```diff
@@ -1,6 +1,105 @@
 {
 	"comment": "",
 	"ignore": "test",
-	"package": [],
+	"package": [
```

Comment:
This file is useful for documenting what we did, but you're right, `go get` ignores this file and just looks at the contents of the vendor directory.